Sources/PrimerSDK/Classes/Constants/Colors.swift

```diff
@@ -0,0 +1,71 @@
+private let darkModeLightGray = UIColor(red: 58/255, green: 58/255, blue: 60/255, alpha: 1)
+private let darkModeRed = UIColor(red: 255/255, green: 69/255, blue: 58/255, alpha: 1)
+private let darkModeBlue = UIColor(red: 10/255, green: 132/255, blue: 255/255, alpha: 1)
+private let darkModeYellow = UIColor(red: 255/255, green: 214/255, blue: 10/255, alpha: 1)
+private let darkModeGreen = UIColor(red: 50/255, green: 215/255, blue: 75/255, alpha: 1)
+
+internal struct Colors {
+    static let Background = UIColor.white
+    
+    struct Text {
+        static let Default = UIColor.primer(lightMode: black, darkMode: darkModeBlack)
+        static let Title = UIColor.primer(lightMode: black, darkMode: darkModeBlack)
+        static let Subtitle = UIColor.primer(lightMode: gray, darkMode: gray)
+        static let AmountLabel = UIColor.primer(lightMode: black, darkMode: darkModeBlack)
+        static let System = UIColor.primer(lightMode: blue, darkMode: darkModeBlue)
+        static let Error = UIColor.primer(lightMode: red, darkMode: red)
+    }
+
+    struct Buttons {
+
+        // Main
+        static let MainDefault = UIColor.primer(lightMode: blue, darkMode: darkModeBlue)
+        static let MainDisabled = UIColor.primer(lightMode: lightGray, darkMode: darkModeLightGray)
+        static let MainSelected = UIColor.primer(lightMode: blue, darkMode: darkModeBlue)
+        static let MainBorderDefault = UIColor.primer(lightMode: blue, darkMode: darkModeBlue)
+        static let MainBorderDisabled = UIColor.primer(lightMode: lightGray, darkMode: darkModeLightGray)
+        static let MainBorderSelected = UIColor.primer(lightMode: blue, darkMode: darkModeBlue)
+        static let MainTextEnabled = UIColor.primer(lightMode: white, darkMode: darkModeWhite)
+        static let MainTextDisabled = UIColor.primer(lightMode: white, darkMode: darkModeWhite)
+
+        // Payment Method
+        static let PaymentMethodDefault = UIColor.primer(lightMode: white, darkMode: darkModeWhite)
+        static let PaymentMethodDisabled = UIColor.primer(lightMode: lightGray, darkMode: darkModeLightGray)
+        static let PaymentMethodSelected = UIColor.primer(lightMode: blue, darkMode: darkModeBlue)
+        static let PaymentMethodBorderDefault = UIColor.primer(lightMode: gray, darkMode: gray)
+        static let PaymentMethodBorderDisabled = UIColor.primer(lightMode: lightGray, darkMode: darkModeLightGray)
+        static let PaymentMethodBorderSelected = UIColor.primer(lightMode: blue, darkMode: darkModeBlue)
+        static let PaymentMethodTextEnabled = UIColor.primer(lightMode: black, darkMode: darkModeBlack)
+        static let PaymentMethodTextDisabled = UIColor.primer(lightMode: gray, darkMode: gray)
+        
+        // Klarna
+        static let KlarnaDefault = UIColor(red: 1, green: 0.702, blue: 0.78, alpha: 1)
+        static let KlarnaTextDefault = UIColor.primer(lightMode: black, darkMode: darkModeBlack)
```

Comment:
[Feedback]
Does that need to be shared? It seems too specific to be in the constants

Reply:
Was probably veering a little close to the Android way of doing things 😅. I'll see if I can reduce the footprint and just reference the raw color constants in the theme.